src/blocks/kdeconnect.rs

```diff
@@ -34,13 +36,14 @@
 //! ```toml
 //! [[block]]
 //! block = "kdeconnect"
-//! format = " $icon {$bat_icon $bat_charge|}{ $notif_icon|} "
+//! format = " $icon {$bat_icon $bat_charge|}{ $notif_icon|}{$network_icon $network |}"
```

Comment:
Format still doesn't look quite right.

You want a ` ` before each `|` and no leading ` ` inside of the `{}`

Like:

`format = " $icon {$bat_icon $bat_charge |}{$notif_icon |}{$network_icon $network |}"`

Reply:
Ok, I also changed the default on the code, not only in the doc. I just didn't want to modify what was already there, but if you wanna change it then it should be correct now.